index.bs

```diff
@@ -3560,8 +3560,8 @@ Maturity Stages on the Recommendation Track</h4>
 				<dd>
 					A Candidate Recommendation Draft
 					is published
-					to solicit review of intended changes from the previous [=Candidate Recommendation Snapshot=].
-					This allows for wider review of the changes
+					to solicit a review of intended changes from the previous [=Candidate Recommendation Snapshot=].
+					This allows for a wider review of the changes
```

Comment:
I would not take this change. We use "wide review" as a set phrase, and it is uncountable. This is not a question of how many reviews we get, just of the fact that the activity of reviewing happens.